onboard/catkin_ws/src/offboard_comms/Arduino Sketchbooks/ThrusterServoArduino/ThrusterServoArduino.ino

```diff
@@ -86,7 +100,7 @@ void loop(){
         servos[i].write(servo_angles[i]);
     }
     pressure_sensor.read();
-    pressure_msg.fluid_pressure = pressure_sensor.pressure(100.0f);
+    pressure_msg.fluid_pressure = pressure_sensor.pressure();
```

Comment:
I believe this is incorrect, the pressure should be in pascals, but the pressure sensor returns data in mbar, so this 100.0f converts to pascal.